src/Compilers/Core/Portable/Operations/IBinaryOperatorExpression.cs

```diff
@@ -32,5 +32,13 @@ public interface IBinaryOperatorExpression : IHasOperatorMethodExpression
         /// value types.
         /// </summary>
         bool IsLifted { get; }
+        /// <summary>
+        /// <code>true</code> if this is a 'checked' binary operator.
+        /// </summary>
+        bool IsChecked { get; }
```

Comment:
i mean, 'Checked' makes sense for statements to me.  After all, we have:

```c#
checked
{
// statements
}
```

That said, i don't feel strongly about it.  Given that checked or not really only affects how an expression will be evaluated at runtime, i think it's fine to only have the bit on the specific expressions.  

(the reason though i did like it on IOPeration was that someone could then simply ask "am i in a checked context or not?" and perhaps make changes as appropriate).